src/ToricVarieties.jl

```diff
@@ -85,61 +124,172 @@ export projective_space
 # 4: Properties
 ######################
 
-function is_normal_variety( v::NormalToricVariety )
+
+"""
+    is_normal_variety( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is normal. (This function is somewhat tautological at this point.)
+
+# Examples
+```julia-repl
+julia> is_normal_variety( projective_space(  2 ) )
+true
+```
+"""
+function is_normal_variety( v::normalToricVariety )
     return true
 end
 export is_normal_variety
 
 
-function is_affine( v::NormalToricVariety )
+"""
+    is_affine( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is affine.
+
+# Examples
+```julia-repl
+julia> is_normal_variety( projective_space(  2 ) )
+false
+```
+"""
+function is_affine( v::normalToricVariety )
     return v.polymakeNTV.AFFINE
 end
 export is_affine
 
 
-function is_projective( v::NormalToricVariety )
+"""
+    is_projective( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is projective, i.e. if the fan of `v` is the the normal fan of a polytope.
+
+# Examples
+```julia-repl
+julia> is_projective( projective_space(  2 ) )
```

Comment:
Very minor nitpick: there's an extra space here (and also in many similar examples). Totally unimportant, though, feel free to ignore :-)
```suggestion
julia> is_projective( projective_space( 2 ) )
```

Reply:
Thank you for pointing this out. Similar strategy as above - new issue (https://github.com/oscar-system/JToric.jl/issues/45) and full resolution very soon.